examples/functions/functions_tester/test/unit/mightFail.test.js

```diff
@@ -0,0 +1,13 @@
+const mightFail = require("../../functions/mightFail");
+
+describe("Recursive retry unit tests", () => {
+  test("should not throw error", async () => {
+    const successRes = await mightFail(3);
+    expect(successRes).toBe(true);
+  });
+
+  test("should throw error", async () => {
+    const failRes = await mightFail(6);
+    expect(failRes).toBe(undefined);
```

Comment:
[nit] Just noticed that we pass the `3` and `6` directly but in `retryHandlers.test.js` we assigned them to variables.